apptools/naming/trait_defs/naming_traits.py

```diff
@@ -73,7 +71,7 @@ def validate(self, object, name, value):
                     self.validate_failed(object, name, value)
             try:
                 value = self._get_binding_for(value)
-            except:
+            except Exception:
```

Comment:
same comment here about the use of `# noqa: EXXX` instead of adding a bare `Exception`